en/docs/chapter_searching/binary_search.md

```diff
@@ -1,24 +1,24 @@
 # Binary search
 
-<u>Binary search</u> is an efficient search algorithm based on the divide-and-conquer strategy. It utilizes the orderliness of data, reducing the search range by half each round until the target element is found or the search interval is empty.
+<u>Binary search</u> is an efficient search algorithm based on the divide-and-conquer strategy. By taking advantage of the orderliness of elements inside the sorted array, it reduces the search interval by half each round until the target element is found or the search interval is empty.
```

Comment:
<u>Binary search</u> is an efficient search algorithm that uses a divide-and-conquer strategy. It takes advantage of the sorted order of elements in an array by reducing the search interval by half in each iteration, continuing until either the target element is found or the search interval becomes empty.

Reply:
accepted